src/state-helper.ts

```diff
@@ -0,0 +1,29 @@
+import * as core from '@actions/core'
+import * as coreCommand from '@actions/core/lib/command'
+
```

Comment:
this is a simple module to make the caller code read more cleanly

this module abstracts the intra-state stuff (pretty simple logic, but reads gross in the other files, so abstracted behind this file instead)